package.json

```diff
@@ -36,16 +36,14 @@
   "repository": "git@github.com:mixmaxhq/search-string.git",
   "devDependencies": {
     "@arethetypeswrong/cli": "^0.17.3",
-    "@mixmaxhq/jest-coverage-config": "^2.2.3",
-    "@mixmaxhq/prettier-config": "^1.0.0",
+    "@mixmaxhq/prettier-config": "^1.1.0",
     "@mixmaxhq/ts-config": "^3.0.1",
-    "@types/jest": "^29.5.13",
     "@types/node": "^22.7.4",
-    "eslint": "^6.8.0",
-    "eslint-config-mixmax": "^4.8.0",
-    "jest": "^29.7.0",
+    "@vitest/coverage-v8": "^3.0.4",
+    "eslint": "^8.57.1",
+    "eslint-config-mixmax": "^5.2.2",
     "prettier": "^1.19.1",
```

Comment:
I hoped to upgrade to the latest Prettier, but `eslint-config-mixmax` uses an outdated version of `eslint-plugin-prettier` that isn't compatible with the latest `prettier`.